src/decision/src/LidarDecision.cpp

```diff
@@ -73,7 +75,7 @@ geometry_msgs::Twist LidarDecision::generate_twist_message(const sensor_msgs::La
 }
 
 std::vector<LidarObstacle> LidarDecision::findObstacles(const sensor_msgs::LaserScan &scan,
-                                                        float max_obstacle_angle_diff) {
+                                                        float max_obstacle_angle_diff, float max_obstacle_distance_diff) {
```

Comment:
we probably want to have `float max_obstacle_distance_diff` on a new line here (same as previous comments)